security/bug-bounty.md

```diff
@@ -1,32 +1,61 @@
+------------------
+Vulnerabilities are eligible for the bug bounty program if they meet a minimum severity threshold. To qualify for a reward under this program, the vulnerability should:
 
-Clever reserves the right to decide if the minimum severity threshold is met and whether it was previously reported. To qualify for a reward under this program, you should:
+* Represent an active threat to the confidentiality or integrity of Clever user data.
+* Be possible to mitigate through standard defensive measures.
+* Include a clear description along with steps to reproduce, including attachments such as screenshots or proof of concept code as necessary.
 
+In general, the following would not meet the threshold for severity:
+* Vulnerabilities on sites hosted by third-parties unless they lead to a vulnerability on the Clever services.
+* Denial of service, spamming, or causing annoyance.
+* Issues with Clever's email server configuration (SPF, DKIM, and DMARC) or name server configuration (DNSSec).
+* Issues that can't be obviously exploited, like self-XSS (an XSS that requires the user to enter a payload that is executed against themselves).
+* Login/logout CSRF.
+* Vulnerabilities affecting users of unsupported or unpatched browsers and platforms.
+
+Clever also reserves the right to deny eligibility for a vulnerability that was previously reported or discovered internally.
 * Be the first to report a specific vulnerability.
-* Send a clear textual description of the report along with steps to reproduce the vulnerability. Include attachments such as screenshots or proof of concept code as necessary.
-* Disclose the vulnerability report directly and exclusively to us. Public disclosure or disclosure to other third parties -- including vulnerability brokers -- before we addressed your report will forfeit the reward.
+* Disclose the vulnerability report directly and exclusively to us. Public disclosure or disclosure to other third parties, including vulnerability brokers, before we address your report will cause you to forfeit the reward.
 
-In general, the following would not meet the threshold for severity:
-* Vulnerabilities on sites hosted by third-parties unless they lead to a vulnerability on the main website
-* Denial of service
-* Spamming
-* Email server configuration (SPF, DKIM, and DMARC)
-* Name server configuration (DNSSec)
-* Self-XSS (an XSS that requires the user to enter a payload)
+Scope
+-----
+The following web services are in scope for this program:
+* https://clever.com (main website)
+* https://schools.clever.com (dashbaord for school districts)
+* https://apps.clever.com (dashboard for applications)
+* https://clever.com/oauth (instant login)
+* https://clever.com/in/ (portal)
+* https://requests.clever.com (districts request access to an app)
 
-Thank you for helping keep Clever safe!
+**Signing up for a school district account**
 
-**Thanks**
+1. Go to the [schools signup](https://schools.clever.com/signup) page.
```

Comment:
@alsmola Can we ask them to sign up at https://clever.com/signup/demo ? That will tag them as a sandbox district automatically.